test_override_builtins failing on master

The master travis python 3 builds are currently failing on `test_override_builtins` (https://travis-ci.org/matplotlib/matplotlib/jobs/240522447):

```python
=================================== FAILURES ===================================

____________________________ test_override_builtins ____________________________

[gw0] linux -- Python 3.6.1 /home/travis/build/matplotlib/matplotlib/venv/bin/python

def test_override_builtins():

        import pylab

    

        ok_to_override = {

            '__name__',

            '__doc__',

            '__package__',

            '__loader__',

            '__spec__',

            'any',

            'all',

            'sum'

        }

    

        # We could use six.moves.builtins here, but that seems

        # to do a little more than just this.

        if six.PY3:

            builtins = sys.modules['builtins']

        else:

            builtins = sys.modules['__builtin__']

    

        overridden = False

        for key in dir(pylab):

            if key in dir(builtins):

                if (getattr(pylab, key) != getattr(builtins, key) and

                        key not in ok_to_override):

                    print("'%s' was overridden in globals()." % key)

                    overridden = True

    

>       assert not overridden

E       assert not True

lib/matplotlib/tests/test_basic.py:41: AssertionError

----------------------------- Captured stdout call -----------------------------

'divmod' was overridden in globals().
```

--- a/lib/matplotlib/tests/test_basic.py
+++ b/lib/matplotlib/tests/test_basic.py
@@ -20,7 +20,8 @@ def test_override_builtins():
         '__spec__',
         'any',
         'all',
-        'sum'
+        'sum',
+        'divmod'
     }
 
     # We could use six.moves.builtins here, but that seems

